doc/README.md

```diff
@@ -11,6 +10,8 @@ respectively. Some documentation is also stored in the library files to be kept
 close to the code it describes. It is included in the manual using the mechanism 
 documented [here](https://www.gap-system.org/Manuals/pkg/GAPDoc-1.5.1/doc/chap4.html).
 
+There is also a document describing [**Changes from Earlier Versions**](Changes.md).
```

Comment:
Please call the file `CHANGES.md` to match conventions used by many other projects.

Reply:
... and let's put it into the root directory, not inside `doc`